### ISSUE TITLE
Unable to get working:

### ISSUE THREAD

**Issue by Francium-Labs** [jw0k/serial2ti83#4]
I am currently not able to get this working. I am attempting to use a Chinese clone Arduino Nano to no success. The program uploads to the board perfectly fine, however neither Ti Connect nor TiLP recognize any device on the comport created by the Arduino. If there's something I am missing please let me know. 

Thank you,
![image_2022-03-01_213211](https://user-images.githubusercontent.com/100744639/156289827-dd274d11-a9cb-45ec-b076-f8f7dd6b1d5f.png)

**Comment by SpeeterYT** [comment]
Same Problem

**Comment by Stoutscientist** [comment]
Did you ever get this to work? I'm also having that issue with my Arduino Uno.

**Comment by Francium-Labs** [comment]
> Did you ever get this to work? I'm also having that issue with my Arduino Uno.

No, unfortunately. I ended up just getting a TI link cable. If anyone does ever figure it out though I'd love to hear what it was.

**Comment by Stoutscientist** [comment]
> > Did you ever get this to work? I'm also having that issue with my Arduino Uno.
> 
> No, unfortunately. I ended up just getting a TI link cable. If anyone does ever figure it out though I'd love to hear what it was.

That's a shame, thanks anyway! If I ever get it working I will update you.

**Comment by meeper123** [comment]
> > > Did you ever get this to work? I'm also having that issue with my Arduino Uno.
> > 
> > 
> > No, unfortunately. I ended up just getting a TI link cable. If anyone does ever figure it out though I'd love to hear what it was.
> 
> That's a shame, thanks anyway! If I ever get it working I will update you.

this is what I did and it worked for me

Here are [all the files](https://drive.google.com/drive/folders/1VbgLqNRsPF2al2W2KnN13rwV-02KUW_q?usp=sharing) make sure to download whole folder and not just the files

- download TILP from sourceforge or the TILP website

- open the ino file in the folder named "USB Serial ti83" and then upload to Arduino uno

- close Arduino ide and open "device manager" (search it up after pushing windows key)

- find the com port of arduino uno (should be "arduino uno (ComXX)")

- right click on com port, select properties, port settings, Advanced, and set com port to COM1 COM2 COM3 or COM4 (make sure there is no in use next to it)

- open TILP, push CTRL+D

- set port to the same number of the com port

- set cable to gray link

- calc as your calculator, click the small magnifying glass, then push OK

- select DIRLIST from top and you should see the calculators files

I hope this helps :)
